rotkehlchen/accounting/pot.py

```diff
@@ -353,8 +355,12 @@ def get_prices_for_swap(
                 asset=asset_in,
                 timestamp=timestamp,
             )
-        except (PriceQueryUnsupportedAsset, NoPriceForGivenTimestamp, RemoteError):
+        except (PriceQueryUnsupportedAsset, RemoteError):
+            in_price = None
+        except NoPriceForGivenTimestamp:
             in_price = None
+            if out_price is None:
+                raise
```

Comment:
I have changed this for the case where `in_price = out_price = None` where the user can take action

Reply:
Hmm but this is unrelated to the PR right? It's a fix for an edge case?

Also can you add a comment explaining this here?

```suggestion
        except NoPriceForGivenTimestamp:
            in_price = None
            if out_price is None:
                raise  # in_price = out_price = None -> notify user
```

Reply:
While fixing the tests I saw that we expect this behaviour. I'v echanged it to only raise the NoPriceForGivenTimestamp when we got no price but we got rate limited